chatgpt_api.py

```diff
@@ -0,0 +1,38 @@
+import os
+import asyncio
+from typing import List, Tuple, Dict
+import openai
+import parsed_presentation
+
+
+async def generate_explain_for_slide(slide_text: str) -> str:
+    """
+    Generate explain from chatGPT
+    :param slide_text: Text of the slide
+    :return: Explain from chatGPT for slide's text
+    """
+    openai.api_key = os.environ.get('OPENAI_API_KEY')
+
+    if len(slide_text.split()) < 10:
```

Comment:
You filter the opening slide with this code? if so, this should have been documented...